VisualStudio.gitignore

```diff
@@ -263,3 +263,6 @@ paket-files/
 # Python Tools for Visual Studio (PTVS)
 __pycache__/
 *.pyc
+
+# Cake
+tools/
```

Comment:
Perhaps we could have this commented out by default, so that users can opt-in to the change...